source/apiVolontaria/location/serializers.py

```diff
@@ -0,0 +1,92 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class CountryBasicSerializer(serializers.ModelSerializer):
+
+    """
+
+    This class represents the Country model serializer.
+
+    A serializer will format the object prior to the interaction with the user.
```

Comment:
This line is not really usefull for developers, It's the same for all Serializers